package.json

```diff
@@ -44,6 +44,7 @@
     "styled-components": ">= 4.X"
   },
   "dependencies": {
+    "acorn": "^6.0.4",
```

Comment:
why do we need to install acorn ?

Reply:
due to `npm install` warning. I'll remove it from this PR to a separate one.